kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgent.java

```diff
@@ -281,6 +293,49 @@ private SslContextFactory getSSLContextFactory() {
         return  sslContextFactory;
     }
 
+    private void startReadinessServer() throws Exception {
+        Server server = new Server();
+        ServerConnector conn = new ServerConnector(server);
+        conn.setPort(HTTP_PORT);
+        server.setConnectors(new Connector[] {conn});
+        ContextHandler readinessContext = new ContextHandler(READINESS_ENDPOINT);
+        readinessContext.setHandler(getReadinessHandler());
+        server.setHandler(readinessContext);
+        server.setStopTimeout(GRACEFUL_SHUTDOWN_TIMEOUT_MS);
+        server.setStopAtShutdown(true);
+        server.start();
```

Comment:
I thought I raised it before, but I cannot see the review comment anywhere. Why do we need a second server here? Why not have one server and use it for everything? I would expect that Jetty should allow to have HTTPS and HTTP ports in one server.